hlces-hamburg.csl

```diff
@@ -0,0 +1,339 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>HLCES (Hamburg)</title>
+    <title-short>HLCES</title-short>
+    <id>http://www.zotero.org/styles/hlces-hamburg</id>
+    <link href="http://www.zotero.org/styles/hlces-hamburg" rel="self"/>
+    <link rel="template" href="http://www1.uni-hamburg.de/COMST/bulletin.html"/>
```

Comment:
That is actually you documentation and therefore you should use `rel="documentation"`.

Did you base your style on another style? Which style did you use as a template?